jwplatform/client.py

```diff
@@ -3,11 +3,12 @@
 import json
 import urllib.parse
 
-from jwplatform import __version__
+from jwplatform import __version__, constants
 from jwplatform.errors import APIError
 from jwplatform.response import APIResponse, ResourceResponse, ResourcesResponse
+from jwplatform.upload import MultipartUpload, determine_upload_method, SingleUpload, UploadType
 
-JWPLATFORM_API_HOST = 'api.jwplayer.com'
+JWPLATFORM_API_HOST = 'api-dev.jwplayer.com'
```

Comment:
```suggestion
JWPLATFORM_API_HOST = 'api.jwplayer.com'
```
Probably want to change this back